### ISSUE TITLE
problem with connected on usb-c port

### ISSUE THREAD

**Issue by tanghxch** [khanhtran-cse/lenovo-ideapad-l340-hackintosh#8]
I have connected my iPhone 11 Pro with l340 laptop on usb-c port but I got error.
it show "USB Devices Disabled" Unplug the device using too much power to re-enable USB devices.
but it appears my device.
now , when i connected with my ipad. it got the same error. and not show anything about my devices.
I already tried on windows 10 it's work.
how to fix that.

**Comment by khanhtrancse** [comment]
Hi @tanghxch 
Do you try other devices?

**Comment by tanghxch** [comment]
> Hi @tanghxch
> Do you try other devices?

Yes , of course. It looks like when i plug with my ipad that use more power than iphone. It will not appear anything. But on the iphone. It appear with the same alert. but show device

**Comment by khanhtrancse** [comment]
I tried and had same error message. I will fix it soon.

**Comment by tanghxch** [comment]
Thank you so much bro.

**Comment by khanhtrancse** [comment]
Hi,

I updated patch to fix this issue. Pay attention to CLOVER/ACPI/patched.

**Comment by artem-tverdokhlebov** [comment]
I tried your UAIC aml, but the message didn't go away, what I'm doing wrong?

**Comment by khanhtran-cse** [comment]
-  Use Hackintool and verify your usb type c port show as usb 3.

For example, SS04 is USB Type C port:
<img width="1055" alt="Screen Shot 2020-05-31 at 21 14 08" src="https://user-images.githubusercontent.com/23311798/83401155-63eabb00-a42e-11ea-955d-3b3059cda78d.png">

**Comment by ivan-holeshau** [comment]
hi. I have connected my ipad 8gen. It is good work
